intranet/apps/bus/consumers.py

```diff
@@ -33,13 +33,18 @@ def receive_json(self, content):  # pylint: disable=arguments-differ
 
         if self.user is not None and self.user.is_authenticated and self.user.is_bus_admin:
             try:
-                route = Route.objects.get(id=content["id"])
-                route.status = content["status"]
-                if route.status == "a":
-                    route.space = content["space"]
-                else:
-                    route.space = ""
-                route.save()
+                if content["time"] == "morning":
+                    route = Route.objects.get(route_name=content["id"])
+                    route.status = content["status"]
```

Comment:
1. Why is `route_name` used instead of `id`? I'd rather keep the consumer the same and tweak the JS.
2. Why does the morning bus code need to be separate? Aside from the use of `route_name` instead of `id`, the only difference is that `route.space` is not set, and that's going to be reset by the Celery task anyway.